doc/nrf/app_boards.rst

```diff
@@ -17,6 +17,15 @@ Board names
 
 The following tables list all boards and build targets for Nordic Semiconductor's hardware platforms.
 
+The build target column uses the following terms for multi-core hardware platform entries:
+
+* ``cpuapp`` -- When you choose this target, you build the application core firmware that is entirely in the Secure Processing Environment (SPE) and does not use :ref:`Trusted Firmware-M (TF-M) <ug_tfm>`.
```

Comment:
Secure Processing Environment as a concept is related to the usage of ArmV8-M Security Extension. I don't think it is completely accurate to call this Secure Processing Environment (SPE)

```suggestion
* ``cpuapp`` -- When you choose this target, you build the application core firmware in a single execution environment, that does not use :ref:`Trusted Firmware-M (TF-M) <ug_tfm>`.
```

Reply:
Rephrased to "* ``cpuapp`` -- When you choose this target, you build the application core firmware as a single execution environment that does not use Cortex-M Security Extensions (:ref:`Trusted Firmware-M (TF-M) <ug_tfm>`)."